aws/influxcloud/Makefile

```diff
@@ -0,0 +1,49 @@
+.PHONY: all \
+	check-for-local-state \
+	configure-state \
+	destroy \
+	plan \
+	plan-destroy
+
+export expected_terraform_version=v0.8
+export actual_terraform_version=$(shell terraform version | head -n 1 | cut -f2 -d" " | cut -d"." -f1,2)
+
+ifneq ($(expected_terraform_version), $(actual_terraform_version))
+  $(error Expected terraform version $(expected_terraform_version).x, but saw $(actual_terraform_version).x)
+endif
+
+export TF_VAR_cluster_address=$(or $(shell PASSWORD_STORE_DIR=~/.registers-pass pass services/influxcloud/admin | tail -n 1), $(error "Couldn't get InfluxCloud cluster address"))
```

Comment:
I think this is right—it's getting the `tail -n1` from the file (which happens to the be the cluster address). The other usage of that credential gets `head -n1`.